master/buildbot/status/web/mybuilds.py

```diff
@@ -11,50 +10,10 @@ class MybuildsResource(HtmlResource):
     @defer.inlineCallbacks
     def content(self, req, cxt):
         master = self.getBuildmaster(req)
-        status = master.getStatus()
-        display_repositories = self.prepare_display_repositories(status)
+        user_id = cxt['authz'].getUserInfo(cxt['authz'].getUsername(req))['uid']
```

Comment:
What happens when user is set through API parameter?

Also, shouldn't the class name be `MyBuildsResource` instead of `MybuildsResource`?

Reply:
This is controller only for MyBuilds page. Controller for api is MyBuildsJsonResource and it handles logged user or user from query params.

I've changed MybuildsResource to MyBuildsResource